### ISSUE TITLE
Printing `created` as UTC.

### ISSUE THREAD

**Issue by davidwessman** [icalendar/icalendar#154]
I am trying to make my iCal-feed get through the http://severinghaus.org/projects/icv/ and I am now stuck at the `CREATED` tag which should be printed in UTC.

Right now I create the events with a certain time-zone and simply setting the:
```ruby
ical_event.created = resource.created_at
ical_event.modified = resource.updated_at
```

In my feed it is printed as:
```
CREATED:20160520T112048
```
compared to for example DTSTAMP
```
DTSTAMP:20160620T115504Z
```

Any clues? Or is this something in implementation?

**Comment by davidwessman** [comment]
I found [this](https://github.com/icalendar/icalendar/blob/master/lib/icalendar/event.rb#L45) and implemented it in my calendar service as well and now it seems to work.
It would however be really cool to do this as default, or show it in documentation somehow.

I could try to prepare either.

**Comment by rahearn** [comment]
Yes, for now that is the way to ensure a UTC timestamp. I'm not super happy with it, but I haven't had time to come up with a better way that will also work whether or not `tzinfo` or `ActiveSupport` are available. Ideas first, and then code would be greatly appreciated if you have any.
